extension/WebJobs.Extensions.RabbitMQ/WebJobs.Extensions.RabbitMQ.csproj

```diff
@@ -1,58 +1,65 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
+    <!-- Common properties (see https://docs.microsoft.com/visualstudio/msbuild/common-msbuild-project-properties) -->
+    <!-- Compiler options (see https://docs.microsoft.com/dotnet/csharp/language-reference/compiler-options) -->
+    <OutputType>Library</OutputType>
     <TargetFramework>netstandard2.0</TargetFramework>
     <AssemblyName>Microsoft.Azure.WebJobs.Extensions.RabbitMQ</AssemblyName>
     <RootNamespace>Microsoft.Azure.WebJobs.Extensions.RabbitMQ</RootNamespace>
-    <ApplicationIcon />
-    <OutputType>Library</OutputType>
-    <StartupObject />
     <SignAssembly>false</SignAssembly>
     <AssemblyOriginatorKeyFile>PublicKey.snk</AssemblyOriginatorKeyFile>
     <DelaySign>true</DelaySign>
+    <LangVersion>latest</LangVersion>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <EnableNETAnalyzers>true</EnableNETAnalyzers>
+    <AnalysisMode>All</AnalysisMode>
+    <EnforceCodeStyleInBuild>true</EnforceCodeStyleInBuild>
+    <!-- <Nullable>enable</Nullable> -->
```

Comment:
Will try to enable [Nullable option](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-options/language#nullable) before publishing the PR.

Reply:
Skipping this change. It would take effort to satisfy nullability checks. Also, there is chance of getting breaking changes introduced.